frontend/public/style/_forms.scss

```diff
@@ -6,3 +6,7 @@ label.co-required:after {
   content: '*';
   padding-left: 3px;
 }
+
+.form-input-padding {
+  padding-bottom: 20px;
```

Comment:
If we should the Bootstrap classes on the form elements, I don't think this should be necessary

Reply:
Well, I am cool just using the spacing with Bootstrap, but I was trying to match the mock ups a little better.

Reply:
form-group solved this issue.